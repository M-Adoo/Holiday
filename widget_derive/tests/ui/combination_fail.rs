use holiday::prelude::*;

#[derive(Widget, CombinationWidget)]
struct A;

#[derive(Widget, CombinationWidget)]
struct B {
  #[proxy]
  a: Checkbox,
  #[proxy]
  b: Checkbox,
}

fn main() { let a = A; }
