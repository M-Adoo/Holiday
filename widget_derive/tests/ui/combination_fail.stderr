error: There is not `#[proxy]` attr specified, required by CombinationWidget
 --> $DIR/combination_fail.rs:4:8
  |
4 | struct A;
  |        ^

error: Too many `#[proxy]` attr specified, need only one
 --> $DIR/combination_fail.rs:7:8
  |
7 | struct B {
  |        ^
